Add some user and tech doc (#3)
vgramer/opa-idea-plugin: docs/devel/sdk/create_rego_file.md

# Create rego file
## Description
This action allows creation of a new `rego file` from a template.

See action usage [here](../../user/features/create_rego_file.md)

## Implementation
The action is implemented by the class [RegoCreateFileAction](../../../src/main/kotlin/org/openpolicyagent/ideaplugin/ide/actions/RegoCreateFileAction.kt)
and registered under the `actions` section in the `plugin.xml`

The default template used to create the file is defined by 2 files:

> *[resources/fileTemplates/internal/Rego File.rego.ft](../../../src/main/resources/fileTemplates/internal/Rego%20File.rego.ft) the template file used to create the rego file

Reviewer: It should be unordered list? Or as it is in one line?

Author: there effectively 2 elements in this unordered list. But syntax was not correct (missing a space between the start and the  bracket). Fixed it :)

# Create rego file
## Description

> This action allow to create a new `rego file` from a template.

Reviewer 2: ```suggestion
This action allows creation of a new `rego file` from a template.
```

See action usage [here](../../user/features/create_rego_file.md)

## Implementation
The action is implemented by the class [RegoCreateFileAction](../../../src/main/kotlin/org/openpolicyagent/ideaplugin/ide/actions/RegoCreateFileAction.kt)

> and register under the `actions` section in the `plugin.xml`

Reviewer 2: ```suggestion
and registered under the `actions` section in the `plugin.xml`
```

> The default template use to create the file is defined by 2 files:

Reviewer 2: ```suggestion
The default template used to create the file is defined by 2 files:
```

*[resources/fileTemplates/internal/Rego File.rego.ft](../../../src/main/resources/fileTemplates/internal/Rego%20File.rego.ft) the template file used to create the rego file

> *[resources/fileTemplates/internal/Rego File.rego.html](../../../src/main/resources/fileTemplates/internal/Rego%20File.rego.html) the description that appear in the template editor (`Settings` ->`Editor` -> `File and Code Templates`)

Reviewer 2: ```suggestion
*[resources/fileTemplates/internal/Rego File.rego.html](../../../src/main/resources/fileTemplates/internal/Rego%20File.rego.html) the description that appears in the template editor (`Settings` ->`Editor` -> `File and Code Templates`)
```
